.eslintrc

```diff
@@ -2,5 +2,9 @@
   "extends": "plugin:shopify/esnext",
   "env": {
     "browser": true
+  },
+  "rules": {
+  	"promise/always-return": 0,
```

Comment:
I don't think this rule has merit in most of the project. It's a common use case to use promise resolution for side effects.